console-extensions.json

```diff
@@ -222,6 +222,48 @@
       "required": ["HIDE_STATIC_PIPELINE_PLUGIN_PIPELINES_LIST"]
     }
   },
+  {
+    "type": "console.page/resource/list",
+    "properties": {
+      "model": {
+        "group": "tekton.dev",
+        "version": "v1beta1",
+        "kind": "PipelineRun"
+      },
+      "component": { "$codeRef": "pipelineRunsList.PipelineRunsListPage" }
+    },
+    "flags": {
+      "required": ["HIDE_STATIC_PIPELINE_PLUGIN_PIPELINERUNS_LIST"]
+    }
```

Comment:
<img width="1425" alt="Screenshot 2024-03-25 at 11 08 45 AM" src="https://github.com/openshift-pipelines/console-plugin/assets/102503482/17413030-2119-4380-96a7-2cfd7303873d">

<img width="1408" alt="Screenshot 2024-03-25 at 11 09 00 AM" src="https://github.com/openshift-pipelines/console-plugin/assets/102503482/5380b089-033e-4d47-aab6-09ee57bec9d1">

In Search menu for PLR, look and feel is different compare to existing design.